### ISSUE TITLE
loss implementation different from paper

### ISSUE THREAD

**Issue by vztu** [chosj95/MIMO-UNet#6]
Hi,

Thanks for sharing this great work~ I found your implementation of loss [here](https://github.com/chosj95/MIMO-UNet/blob/4a523544e3c5192f5305610b761ee3a4d5d74648/train.py#L54-L69) is different from the paper, where the "1/t_k" normalizationweight is not used in your code. Could you help clarify this?

**Comment by chosj95** [comment]
Hi,

Thanks for your interest in our work.
We use default ```torch.nn.L1Loss()``` for our loss function [link](https://github.com/chosj95/MIMO-UNet/blob/4a523544e3c5192f5305610b761ee3a4d5d74648/train.py#L13). 

In our knowledge, the default mode for ```reduction``` of L1 loss in Pytorch is ```mean``` [link](https://pytorch.org/docs/stable/generated/torch.nn.L1Loss.html).

The following is the toy example for you.

```
>>> import torch
>>> l = nn.L1Loss()

>>> x = torch.randint(0,5,(3,5),dtype=torch.float)
>>> x
tensor([[2., 4., 4., 1., 2.],
        [4., 4., 2., 3., 3.],
        [2., 4., 3., 1., 4.]])
>>> y = torch.randint(0,5,(3,5),dtype=torch.float)
>>> y
tensor([[4., 4., 1., 3., 1.],
        [0., 1., 1., 4., 0.],
        [2., 2., 4., 0., 3.]])
>>> l(x,y)
tensor(1.6667)

```

**Comment by vztu** [comment]
Ahh yes sorry I think I misunderstood the `l1_norm` here. The `|| x ||_1` operator only does summation without averaging. Thank you for clarifying this!
